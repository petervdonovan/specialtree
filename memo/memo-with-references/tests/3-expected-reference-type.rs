use memo_with_references::memo;

#[memo(cache)]
fn memome(cache: String) {}

fn main() {}
