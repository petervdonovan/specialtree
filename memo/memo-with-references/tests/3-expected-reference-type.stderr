error: expected reference type with an explicit lifetime
 --> tests/3-expected-reference-type.rs:4:18
  |
4 | fn memome(cache: String) {}
  |                  ^^^^^^
